### PR TITLE
add connection status as filter - fixes #142

### LINKED ISSUE #142: the `probes` command could use a `--status=` filter and a few others
I was trying to find my probe on the cli today and I immediately tried to filter out the noise with a `--status connected` and it barked at me that this wasn't supported.  It'd be nice if we could filter the list a little better with some tags like these:

    --id
    --id-greater-than
    --id-less-than
    --address_v6
    --is-anchor
    --country
    --region (this would be nice, but it's obviously problematic)
    --is-public
    --status
    --in-prefix
    --in-asn

### PR DESCRIPTION
**Pull request by astrikos**

### CODE REVIEW

File: ripe/atlas/tools/commands/probes.py [hunk 1]
```diff
@@ -166,6 +166,15 @@ def add_arguments(self):
                 "command."
             )
         )
+        self.parser.add_argument(
+            "--status",
+            type=int,
+            choices=[0, 1, 2, 3],
```
Comment (danielquinn) [inline ripe/atlas/tools/commands/probes.py]: It would be nice to have the option to say `--status connected` rather than `--status 1`.  As it is, this isn't very user-friendly.

Comment (danielquinn) [inline ripe/atlas/tools/commands/probes.py]: Ooh, or better yet `--statuses never-connected,disconnected,abandoned` would be nice too.

Comment (danielquinn) [inline ripe/atlas/tools/commands/probes.py]: Though I suppose you could support comma-separated values for `--status` and that'd be totally ok.  No need to implement `--status` *and* `--statuses`.

Comment (astrikos) [inline ripe/atlas/tools/commands/probes.py]: yes i started doing that in the beginning but then i saw i had to do translation stuff to pass it to api (capital first letter and never connected has a space), so i said naah. also the multiple values is not supported on the API, so it would be doable on the cli without multiple queries.
So i think I will leave it as it is and someone can prettify it later when we add the multiple status feature on the api

Comment (danielquinn) [inline ripe/atlas/tools/commands/probes.py]: Ah well if the API doesn't support multiple values, yeah, let's not do that.  I'd still like to see human-friendly words for this, but I'm happy to do that as a separate PR at some point.

**Comment by astrikos** [comment]
ok merging that one and will do next one in diff PR before releasing.
